app.js

```diff
@@ -44,25 +47,74 @@ app.Views.TodoAppView = Backbone.View.extend({
         this.todoContent.append(this.subviews.todoListView.render());
 
         return this.$el;
+    },
+    createTodoOnEnter: function(e) {
+        const title = this.newTodoField.val().trim();
+        if (e.which == 13 && title != "") {
+
+            this.subviews.todoListView.collection.add(new app.Models.TodoItem({
+                title: title
+            }));
+        }
     }
 });
 app.Views.ToDoItemView = Backbone.View.extend({
     tagName: "li",
     className: "todo-item",
+    initialize: function() {
+        // this.listenTo(this.model, "destroy", this.remove);  // is this needed even though list view already listens for remove
```

Comment:
Please address the comment here.

Do I need to listenTo to model.destroy and this.remove in response? Technically, the collection will already do this by listening to "remove" and invoking removeSubviews()